File: 2017/2/kk.c
#include <stdio.h>
#include <unistd.h>
#include <string.h>

int main(){
	char buffer[1024];
	unsigned int sum = 0;

	while (fgets(buffer, sizeof(buffer), stdin)) {
		char *b = buffer;
		unsigned int min = 0xffffffff;
		unsigned int max = 0;
		unsigned int v;
		while (sscanf(b, "%d", &v)) {
			if (v < min)
				min =v;
			if (v > max)
				max =v;
			b = strstr(b, "\t");
			if (!b)
				break;
			b++;
		}
		sum += max - min;
	}

	printf("%ld\n", sum);
}

File: 2017/2/kk.py
import sys

n_sum = 0
for line in sys.stdin:
    l = line.split()
    done = False
    for a in range(len(l)):
        if done:
            break
        v1 = int(l[a])
        for b in range(a + 1, len(l)):
            v2 = int(l[b])
            M = max(v1,v2)
            m = min(v1,v2)
            if (M % m) == 0:
                n_sum += M / m
                done = True
                break
print n_sum
